guides/VERSIONING.md

```diff
@@ -0,0 +1,15 @@
+Version control of Repositories
+===============================
+
+Following [Semantic Versioning 2.0.0](https://semver.org/).
+
+* When creating release branches or tags it is our standard to not use the patch version (`1.0.0 = Major.Minor.Patch`)
+
+* Any new release will force the version to go up by either a minor or major version, see examples below:
+
+```
+Current version is 1.1.0
+
+* Minor release updates version to 1.2.0
+* Major release updates version to 2.0.0
+```
```

Comment:
It is likely also worth putting a note to say that we use `1.0.0` as the first stable release of a repo.